examples/extensions/remove_entire_sitelink_campaign_extension_setting.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Removes the entire sitelink campaign extension setting.
+
+Removes both the sitelink campaign extension setting itself and its associated
+sitelink extension feed items. This requires two steps, since removing the
+campaign extension setting doesn't automatically remove its extension feed
+items.
+
+To make this example work with other types of extensions, replace
+ExtensionType.SITELINK with the extension type you wish to remove.
+"""
+
+
+import argparse
+import sys
+
+from google.api_core import protobuf_helpers
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+from google.ads.google_ads.util import ResourceName
+
+
+def main(client, customer_id, campaign_id):
+    """The main method that creates all necessary entities for the example.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID.
+        campaign_id: the campaign ID.
+    """
+    # Initialize an array of MutateOperations
+    mutate_operations = []
+    sitelink_campaign_extension_setting_mutate_operation = _create_sitelink_campaign_extension_setting_mutate_operation(
+        client, customer_id, campaign_id
+    )
+    mutate_operations.extend([sitelink_campaign_extension_setting_mutate_operation])
```

Comment:
why coerce `sitelink_campaign_extension_setting_mutate_operation` into a list when you can just use `.append` in this case?

Reply:
No good reason at all. Fixed!